backend/src/controllers/Students.js

```diff
@@ -0,0 +1,64 @@
+const Student = require('../models/Student')
+const validateStudent = require('../validation/Student')
+
```

Comment:
Could we add a getAll endpoint as well? Will be super useful for future admin site

Reply:
okay, I have added the getAll endpoint